src/rust/engine/fs/store/src/local.rs

```diff
@@ -48,6 +57,8 @@ impl ByteStore {
     let root = path.as_ref();
     let files_root = root.join("files");
     let directories_root = root.join("directories");
+    let large_files_root = root.join("immutable");
```

Comment:
This should probably be next to the `lmdb_store` directory, rather than inside of it. And all of the content stored in LMDB is `immutable`, so probably something like "large_$x" ... "large blobs", "large files", "large content", etc.

Reply:
I thought about this when making the change. The reason its this way is because the "`lmdb_store`" directory isn't an "LMDB Store" from the user's perspective. It's the "Local Store" whose default value happens to be named "`lmdb_store`". So I think this DOES belong in the "local store" regardless of the (now bad) name.

thoughts?